### DISCUSSION TITLE
KeePassXC Branded Merchandise

### DISCUSSION THREAD

**Discussion by OLLI-S** [keepassxreboot/keepassxc#4130] [Q&A]
## Summary
Please offer merchandise for KeePassXC

## Details
I really would love to have a sticker of KeePassXC on my business notebook and my private Surface Go.
Both have not yet any stickers on it.

There are other products that could be interesting, like:

- coffee mugs
- coffee cups
- T-Shirts
- Hoodies

I am not sure how many people would buy this.
But maybe there is a merchandise service where you upload and create the products the logo and only when a user wants to order an item it is produced.
So you don't have any costs and merchandise article on hold.

I know I am totally crazy, but I really love KeePassXC.

**Comment by droidmonkey** [comment]
I purchased stickers a while ago and distributed them to my Patreon patrons. It was an incentive to donate and become part of our journey. Setting up a general store is possible, we could do it on cafepress or similar. I'll look into it, not a bad idea.

**Comment by droidmonkey** [comment]
Some options I found almost immediately:

- https://www.stickers.pub/sell.html
- https://www.redbubble.com/about/selling
- https://www.spreadshop.com/sell-merchandise

**Comment by 4jNsY6fCVqZv** [comment]
If you set up a shop, I would ask you to make sure that the products are produced sustainably, i.e. ecologically and under fair working conditions. Ideally, these would be recycled products that are Cradle to Cradle certified and can either be kept in the cycle (e.g. by repair) or simply returned to the biological cycle (e.g. 100% compostable products).
More about Cradle to Cradle: https://en.wikipedia.org/wiki/Cradle-to-cradle_design
I think this ethical approach also fits very well with the Open Source idea.
Could you imagine that?

**Comment by droidmonkey** [comment]
If such a service exists that provides that certification or assurance, it will be considered.  Although I would argue if you are concerned with environmental impact of "tchotchkes", simply don't buy one in the first place. Then nothing will be wasted and no unfair labor will occur.

**Comment by 4jNsY6fCVqZv** [comment]
Exactly, in a world of climate change I am very concerned about this. I agree with you: not buying at all would also be the best choice from my point of view. If sustainability criteria are not met, it would also be better for me to stop producing and selling them.
If people are considering it anyway, I suggest they pay attention to the impact of the product. And as a supplier you can influence what is offered branded in your shop.

Related to Cradle-to-cradle products: I think there are offers. Probably a matter of research. At least there are sustainable and fair products. Cradle-to-cradle is probably the ideal solution. And for that, it should be clear which products should be in the shop. 

However, I lower my thumb for this issue, because a better world is important to me and I know that we all can contribute something to achieve it. And if only that means: Don't bring more stickers(garbage) and hoodies full of harmful substances from child labour into the world.

The comment does not change my opinion about your software. Thanks for your great work!

**Comment by OLLI-S** [comment]
My intention was to make KeePassXC more popular so more people use it (instead of KeePass).

**Comment by 4jNsY6fCVqZv** [comment]
Thanks, I'm sure you have good intentions.
When I read your argument, I ask myself: Couldn't this be achieved by personal conversations about good password managers as well?
Another thought: I think far too few people use password managers at all. Is it really significant that people who already have security awareness will use KeePassXC instead of KeePass in the future?
Or should people be brought more into an awareness where they start to care more about the protection of their data and privacy? And who out of this changed attitude start using something like an open source password manager at all.

**Comment by OLLI-S** [comment]
I work in a very huge company as usability engineer, so I have many projects where I inspect their applications. I think when I have a sticker on my laptop at work then some people will ask what KeePassXC is and this is a "door opener".
And such a sticker should not be more than 10 USD, so I can afford this easily...

**Comment by GlenCooper** [comment]
I could use a couple more stickers for my laptop.  I'd gladly stick a KeePassXC sticker on there if I could pay for it using the Bitcoin Lightning Network.

There's still some space left;
https://drive.google.com/open?id=1ujduO9Afk4sja2GQGpDS6YvhgJYiwVM6
![20200121T2347Z-laptop](https://user-images.githubusercontent.com/967793/72853325-f755f780-3ca8-11ea-875b-732787c3bd8e.JPG)

**Comment by OLLI-S** [comment]
So, will stickers be available?

**Comment by GlenCooper** [comment]
> Some options I found almost immediately:
> 
>     * https://www.stickers.pub/sell.html
> 
>     * https://www.redbubble.com/about/selling
> 
>     * https://www.spreadshop.com/sell-merchandise

I looked thru these briefly.  https://www.spreadshop.com/sell-merchandise looks promising.  Does KeePassXC have any high res logos available?

**Comment by droidmonkey** [comment]
We have SVG everything so you can get whatever res you want

**Comment by wolframroesler** [comment]
Here's what I came up with, took me ten minutes of clicking and zero creativity (I'm not a designer): https://teespring.com/shop/keepassxc

**Comment by droidmonkey** [comment]
LOL, thumbs up for the keepassxc leggings

**Comment by OLLI-S** [comment]
ROFL, the leggings......
I have legs like a deer: not that slim and elegant but as hairy.....
:stuck_out_tongue:

**Comment by phoerious** [comment]
I think that needs another font at least. :-D

**Comment by wolframroesler** [comment]
So the plan is:

1. Advertise this really really hard
1. Make tons of $$$ from it
1. Use the money to fund #432.

And maybe 0. put some more effort into the design first.
